test/index.test.ts

```diff
@@ -48,8 +48,17 @@ describe("Approveman tests", () => {
 
   beforeEach(() => {
     nock.disableNetConnect();
-    probot = new Probot({ id: 123, cert: mockCert });
-    probot.load(approvemanApp);
+    probot = new Probot({
+      id: 1,
+      githubToken: "test",
```

Comment:
Codacy found an issue: [The key 'githubToken' is not sorted alphabetically](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6106171)